oxidation/libparsec/crates/crypto/Cargo.toml

```diff
@@ -15,17 +15,25 @@ base32 = "0.4.0"
 
 sodiumoxide = { version = "0.2.7", optional = true }
 libsodium-sys = { version = "0.2.7", optional = true }
+openssl = { version = "0.10", optional = true, features = ["vendored"] }
+openssl-sys = { version = "0.9", optional = true }
+zeroize = { version = "1", optional = true, features = ["alloc"]}
```

Comment:
This is a cool idea !

we wanted to do zeroizing of keys, but on Python memory management makes it complex to implement reliably.

I've created https://github.com/Scille/parsec-cloud/issues/3791 to see if we can use zeroize more broadly in crypto